tests/callbacks/test_pruning.py

```diff
@@ -198,7 +194,6 @@ def test_with_pruning_callback_misconfiguration(tmpdir):
         _ = ModelPruning(**model_pruning_args)
 
 
-@pytest.mark.skipif(not _PYTORCH_PRUNE_AVAILABLE, reason="PyTorch prung is needed for this test. ")
```

Comment:
why prune edit in this PR?

Reply:
Because noticed with the addition of `PYTORCH_GREATER_THAN_1_6`.

I can move it out if you want